pkg/sip/inbound.go

```diff
@@ -548,19 +548,19 @@ func (c *inboundCall) runMediaConn(offerData []byte, conf *config.Config, featur
 		EnableJitterBuffer:  c.s.conf.EnableJitterBuffer,
 	}, RoomSampleRate)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("cannot create media port: %w", err)
```

Comment:
Should we use some error wrapping utility or psrpc.NewError here (and below) instead, to allow extracting the original error?